app/services/candidate_interface/can_add_course_choice.rb

```diff
@@ -0,0 +1,16 @@
+module CandidateInterface
+  class CanAddCourseChoice
+    def self.can_add_course_choice?(application_form:)
```

Comment:
if we use a feature flag i'll add something like `return true unless FeatureFlag.active('end_of_cycle')`